### PR TITLE
Add Spanish translation for migration.md 

### PR DESCRIPTION
**Pull request by jacostaf10**
Added Spanish translation for `migration.md` in `fluent` folder

### CODE REVIEW

File: docs/fluent/migration.es.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Migraciones
+
+Las migraciones son una especie de control de versiones para tu base de datos. Cada migración define un cambio en la base de datos y cómo deshacerlo. Modificando tu base de datos mediante migraciones creas una manera consistente, testable y fácil de compartir de evolucionar tu base de datos a lo largo del tiempo.
```
Comment (alemohamad) [inline docs/fluent/migration.es.md:3]:
```suggestion
Las migraciones son una especie de control de versiones para tu base de datos. Cada migración define un cambio en la base de datos y cómo deshacerlo. Modificando tu base de datos mediante migraciones creas una manera consistente, testable y fácil de compartir, de evolucionar tu base de datos a lo largo del tiempo.
```

File: docs/fluent/migration.es.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+```swift
+// Una migración de ejemplo.
+struct MyMigration: Migration {
+    func prepare(on database: Database) -> EventLoopFuture<Void> {
+        // Haz un cambio en la base de datos.
+    }
+
+    func revert(on database: Database) -> EventLoopFuture<Void> {
+    	// Deshaz el cambio hecho en `prepare`, si es posible.
```
Comment (alemohamad) [inline docs/fluent/migration.es.md:13]:
```suggestion
        // Deshaz el cambio hecho en `prepare`, si es posible.
```

File: docs/fluent/migration.es.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Si estás usando `async`/`await` deberías implementar el protocolo `AsyncMigration`:
+
+```swift
+struct MyMigration: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        // Haz un cambio en la base de datos.
+    }
+
+    func revert(on database: Database) async throws {
+    	// Deshaz el cambio hecho en `prepare`, si es posible.
```
Comment (alemohamad) [inline docs/fluent/migration.es.md:27]:
```suggestion
        // Deshaz el cambio hecho en `prepare`, si es posible.
```

File: docs/fluent/migration.es.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+```sh
+swift run App serve --auto-migrate
+```
+
+También puedes hacerlo de manera programática. 
+
+```swift
+try app.autoMigrate().wait()
+
+// or
```
Comment (alemohamad) [inline docs/fluent/migration.es.md:88]:
```suggestion
// o
```

**Review by alemohamad — CHANGES_REQUESTED** [review]

File: docs/fluent/migration.es.md (hunk 2 above)
Comment (jacostaf10) [reply docs/fluent/migration.es.md:13]: I don't understand the purpose of this change. Did I miss a tab on the comment?

**Review by alemohamad — APPROVED** [review]
This looks great! 👍

**Pull request merged**